test/test_regenerator.rb

```diff
@@ -0,0 +1,193 @@
+require 'helper'
+
+class TestRegenerator < Test::Unit::TestCase
+  context "The site regenerator" do
+    setup do
+      FileUtils.rm_rf(source_dir(".jekyll-metadata"))
+
+      @site = Site.new(Jekyll.configuration({
+        "source" => source_dir,
+        "destination" => dest_dir,
+        "collections" => {
+          "methods" => {
+            "output" => true
+          }
+        }
+      }))
```

Comment:
I added a `fixture_site` somewhere or have a means of generating this without touching the file system. Somewhere in `test/helper.rb`. Could you please use that?